### PR TITLE
Graphql search

### PR DESCRIPTION
**Pull request by jstucke**

### CODE REVIEW

**Review by dorpvom — CHANGES_REQUESTED** [review]

**Comment by dorpvom** [comment]
Also: Where are the tests?

**Comment by jstucke** [comment]
> Also: Where are the tests?

a test file was missing (see https://github.com/fkie-cad/FACT_core/pull/1269/commits/4b527866cd2a80081287801654be27dcbdb4f09a)

File: src/storage/graphql/hasura/restart.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import sys
```
[the remark this answers is not in the document]
Comment (jstucke) [reply src/storage/graphql/hasura/restart.py:9]: added

File: src/storage/graphql/hasura/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    restart: always
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    environment:
+      HASURA_GRAPHQL_METADATA_DATABASE_URL: postgres://postgres:postgrespassword@postgres:5432/postgres
+      PG_DATABASE_URL: postgres://postgres:postgrespassword@postgres:5432/postgres
+      HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
+      HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup, http-log, webhook-log, websocket-log, query-log
+      HASURA_GRAPHQL_CONSOLE_ASSETS_DIR: /srv/console-assets
+      HASURA_GRAPHQL_METADATA_DEFAULTS: '{"backend_configs":{"dataconnector":{"athena":{"uri":"http://data-connector-agent:8081/api/v1/athena"},"mariadb":{"uri":"http://data-connector-agent:8081/api/v1/mariadb"},"mysql8":{"uri":"http://data-connector-agent:8081/api/v1/mysql"},"oracle":{"uri":"http://data-connector-agent:8081/api/v1/oracle"},"snowflake":{"uri":"http://data-connector-agent:8081/api/v1/snowflake"}}}}'
```
[the remark this answers is not in the document]
Comment (jstucke) [reply src/storage/graphql/hasura/docker-compose.yaml]: removed

**Review by dorpvom — APPROVED** [review]

**Pull request merged**